config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1599,6 +1599,9 @@ presubmits:
     labels:
       preset-service-account: "true"
       preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+      preset-pull-kubernetes-e2e: "true"
+      preset-pull-kubernetes-e2e-gce: "true"
     spec:
       containers:
       - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221107-33c989e684-master
```

Comment:
Not 100% sure but do we need to add `--repo=k8s.io/release` similar as here (https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L35C16-L36)?